app/src/main/res/values-de/strings-de.xml

```diff
@@ -204,10 +204,10 @@
 	<string name="wear_integration">Wear Integration</string>
 	<string name="send_data_to_android_wear_watch">Daten an Watchface senden</string>
 	<string name="android_wear_integration">Android Wear Integration</string>
-	<string name="pref_wear_connectG5">Schließen Sie Wear bis G5</string>
-	<string name="pref_summary_wear_connectG5">Schließen Sie tragen direkt an G5</string>
-	<string name="pref_use_wear_connectG5">Verwendung Tragen G5 Collection Service</string>
-	<string name="pref_summary_use_wear_connectG5">Force-Telefon Wear G5 Collection Service zu verwenden,</string>
+	<string name="pref_wear_connectG5">Verbinde die Uhr mit dem G5</string>
+	<string name="pref_summary_wear_connectG5">Direkte Verbindung zum G5</string>
+	<string name="pref_use_wear_connectG5">Verwende den G5 Kollektor-Dienst</string>
+	<string name="pref_summary_use_wear_connectG5">Zwinge das Telefon, den Wear-Collection-Dienst zu benutzen</string>
```

Comment:
Ha, you found some failed google translations :+1: 
But I can't find them in the general strings.xml. I guess (hope!) those have never been used :)